### PR TITLE
Fix more flutter tutorial things

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: docs/tutorials/control/flutter-app.md [hunk 1]
```diff
@@ -509,8 +526,8 @@ class _LocationScreenState extends State<LocationScreen> {
   }
 
   void _navigateToRobot(Robot robot) {
-    Navigator.of(context).push(
-        MaterialPageRoute(builder: (_) => RobotScreen(widget._viam, robot)));
+    Navigator.of(context)
+        .push(MaterialPageRoute(builder: (_) => LocationScreen(_viam, location)));
```
Comment (sguequierre) [inline docs/tutorials/control/flutter-app.md:530]: Why change this? I haven't in the version I'm using. This enables one to nav from location screen to a RobotScreen()

File: docs/tutorials/control/flutter-app.md [hunk 2]
```diff
@@ -422,6 +422,23 @@ Follow the steps below to get your API key and create an environment variables f
      }
    ```
 
+8. Reopen your <file>pubspec.yaml</file> file and paste the following two lines at the end of it, inside the `flutter:` section.
```
Comment (sguequierre) [inline docs/tutorials/control/flutter-app.md:425]: (opt) You could probably combine this into just one instruction

**Review by sguequierre — CHANGES_REQUESTED** [review]

File: docs/tutorials/control/flutter-app.md (hunk 2 above)
Comment (JessamyT) [reply docs/tutorials/control/flutter-app.md:425]: Sorry I don't understand. What do you mean?

File: docs/tutorials/control/flutter-app.md (hunk 1 above)
Comment (JessamyT) [reply docs/tutorials/control/flutter-app.md:530]: Ooh yeah you're right sorry I was meaning to make something more consistent but messed this part up. Reverting!

File: docs/tutorials/control/flutter-app.md (hunk 2 above)
Comment (sguequierre) [reply docs/tutorials/control/flutter-app.md:425]: It's hard to understand where to put the first code block without the second, so you could probably just have the second code block!

**Review by sguequierre — APPROVED** [review]

**Pull request merged**